models/issue.go

```diff
@@ -246,6 +246,7 @@ func (issue *Issue) APIFormat() *api.Issue {
 
 	apiIssue := &api.Issue{
 		ID:       issue.ID,
+		URL:      issue.HTMLURL(),
```

Comment:
We don't support a separate api url yet, so just take the regular url and append the api path